src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -156,11 +156,8 @@
   <data name="FilePathTooLongWarning" xml:space="preserve">
     <value>The file '{0}' path, name, or both are too long. Your package might not work without long file path support. Please shorten the file path or file name.</value>
   </data>
-  <data name="IconUrlAndIconWarning" xml:space="preserve">
-    <value>Both 'iconUrl' and 'icon' elements were detected. Consider removing the 'iconUrl' element since it will be deprecated. To learn more, go to https://aka.ms/deprecateIconUrl.</value>
-  </data>
   <data name="IconUrlDeprecationWarning" xml:space="preserve">
-    <value>The 'iconUrl' element will be deprecated. Consider using the 'icon' element instead.</value>
+    <value>The 'PackageIconUrl'/'iconUrl' element is deprecated. Consider using the 'PackageIcon'/'icon' element instead.</value>
```

Comment:
Are you missing: https://aka.ms/deprecateIconUrl maybe?

Reply:
Good catch. I will add it.